index.html

```diff
@@ -2238,14 +2263,10 @@ <h2>The <dfn>record type</dfn> string</h2>
       Attempt to <a>abort a pending write operation</a>.
     </li>
     <li>
-      Stop the <a>dispatch NFC content</a> steps.
-    </li>
-    <li>
-      Clear the <a>pending read tuple</a>.
+      Attempt to <a>abort a pending scan operation</a>.
```

Comment:
I think we should also abort steps below when promise gets rejected right?

Reply:
I don't understand.